fe/fe-core/src/main/java/org/apache/doris/analysis/CreateTableStmt.java

```diff
@@ -519,19 +517,23 @@ public void analyze(Analyzer analyzer) throws UserException, AnalysisException {
             }
 
             if (columnDef.getType().isObjectStored()) {
-                hasObjectStored = true;
-                objectStoredColumn = columnDef.getName();
+                if (columnDef.getType().isBitmapType()) {
+                    if (keysDesc.getKeysType() == KeysType.DUP_KEYS) {
+                        throw new AnalysisException("column:" + columnDef.getName()
+                                + " must be used in AGG_KEYS or UNIQUE_KEYS.");
+                    }
+                } else {
+                    if (keysDesc.getKeysType() != KeysType.AGG_KEYS) {
+                        throw new AnalysisException("column:" + columnDef.getName() + " must be used in AGG_KEYS.");
+                    }
+                }
             }
```

Comment:
This pr also changed the behavior of hll and quantile_state, should make corresponding changes to their documents, and add case

Reply:
> This pr also changed the behavior of hll and quantile_state, should make corresponding changes to their documents, and add case

the behavior of hll and quantile_state is not changed, only bitmap type is allowed to appear in unique table.